app/services/sponsored-content.js

```diff
@@ -0,0 +1,54 @@
+import Service, { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+export default Service.extend({
```

Comment:
can we extract this service to design-system repo, so it can be used in search recirc module as well?